tests/similarity_methods/test_quasi_local.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from social_network_link_prediction.similarity_methods import quasi_local_similarity
+from timeout_decorator import timeout
+from tests import Configs
+from time import time
+
+
+class TestQuasiGlobalSimilarityMethods(unittest.TestCase):
+
+    def setUp(self):
+        self.start_time = time()
+
+    def tearDown(self):
+        t = time() - self.start_time
+
+        print(f"{round(t, 2)} s")
+
+    @unittest.skip("Non è presente in scikit-network")
+    def test_lpi_1(self):
+        pass
+
+    @unittest.skip("Non è presente in scikit-network")
+    def test_lpi_2(self):
+        pass
+
+    @unittest.skip("Non è presente in scikit-network")
+    def test_lpi_3(self):
+        pass
+
+    # TODO: @ncvesvera @fagiolo ricontrollare
+    @timeout(Configs.timeout)
+    def test_lpi_time(self):
+        g, adjacency = Configs.load_hard_dataset()
+        epsilon = .1
+        n = 4
+
+        res = quasi_local_similarity.local_path_index(g, epsilon, n)
+        # print(res)
+
+        self.assertIsNotNone(res)
```

Comment:
@ncvescera @F-a-b-r-i-z-i-o ricontrollare i parametri che vanno passati al metodo `local_path_index`

Reply:
Risolto in 9328b9e